docs/lib/datastore/fragments/flutter/sync/10-installPlugin.md

```diff
@@ -0,0 +1,16 @@
+### Add the API plugin
+
+DataStore's cloud synchronization uses the [API category](~/lib/graphqlapi/getting-started.md) behind the scenes. Therefore, the first step is to add an API plugin.
+
+Make sure that you declare a dependency on the DataStore plugin in your app-level pubspec.yaml:
```

Comment:
Are there multiple `pubspec.yaml`, where "app-level" is a useful distinguishment?
```suggestion
Make sure that you declare a dependency on the DataStore plugin in your app-level `pubspec.yaml`:
```

Reply:
I fixed this.